src/components/Main.js

```diff
@@ -87,17 +83,13 @@ export default class Main extends React.Component {
   }
 }
 
-function injectCSS (url) {
-  var link = document.createElement('link');
-  link.href = url;
-  link.type = 'text/css';
-  link.rel = 'stylesheet';
-  link.media = 'screen,print';
-  link.setAttribute('data-aframe-editor', 'style');
-  document.getElementsByTagName('head')[0].appendChild(link);
-}
-
 (function init () {
+  var webFont = document.createElement('script');
+  webFont.src = 'https://ajax.googleapis.com/ajax/libs/webfont/1.6.16/webfont.js';
+
+  var webFontLoader = document.createElement('script');
+  webFontLoader.innerHTML = "WebFont.load({google: {families: ['Roboto Mono']}});";
```

Comment:
Ok, I think the problem is that you've removed also the `font-awesome` font, which is needed for the icons in the scenegraph, I was trying your branch and I couldn't get the icons on it so I thought the dependency was failing